pytensor/scalar/basic.py

```diff
@@ -1895,10 +1895,14 @@ def L_op(self, inputs, outputs, gout):
         return (gx, gy)
 
 
-scalar_maximum = ScalarMaximum(upcast_out, name="maximum")
+maximum = Maximum(upcast_out, name="maximum")
```

Comment:
no need for custom names, also remove for any other ops where name is same as class. remove scalar_prefix from names like scalar_softplus as well